test/lost-plugin-options.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const postcss = require('postcss');
+const lost = require('../lost');
+const expect = require('chai').expect;
+
+describe('plugin-options', () => {
+  it('set options with plugin options', async () => {
+    const input = `
+    div {
+      lost-column: 1/2;
+    }
+    `;
+    const output = `
+    div {
+      flex-grow: 0;
+      flex-shrink: 0;
+      flex-basis: calc(99.9% * 1/2 - (15px - 15px * 1/2));
+      max-width: calc(99.9% * 1/2 - (15px - 15px * 1/2));
+      width: calc(99.9% * 1/2 - (15px - 15px * 1/2));
+    }
+    div:nth-child(1n) {
+      margin-right: 15px;
+      margin-left: 0;
+    }
+    div:last-child {
+      margin-right: 0;
+    }
+    div:nth-child(2n) {
+      margin-right: 0;
+      margin-left: auto;
+    }
+    `;
+    const result = await postcss([
+      lost({
+        gutter: '15px',
+        flexbox: 'flex',
+        cycle: 'auto'
+      })
+    ])
+      .process(input);
+      expect(output).to.equal(result.css);
```

Comment:
```suggestion
      expect(result.css).to.equal(output);
```
It's minor, but this reads better when reading errors.

What it should read like when this test fails:
<img width="435" alt="Screen Shot 2022-06-29 at 21 56 23" src="https://user-images.githubusercontent.com/3523268/176595954-57e7f909-b087-467c-810b-d3b4695d15d6.png">